disco_aws_automation/disco_subnet.py

```diff
@@ -0,0 +1,322 @@
+"""
```

Comment:
Oh look, this is the package I thought should exist!  Maybe?  It's already pretty big, even without any of the possibly-excess stuff from disco_aws... :-/